gpt3.py

```diff
@@ -78,11 +81,16 @@ async def gpt3(client: Client, message: Message):
             else:
                 await message.edit_text("No valid 'text' value found in the response.")
         else:
-            await message.edit_text(
-                f"Request failed with status code {response.status_code}"
-            )
+            try:
+                response = await RendyDevChat.chat_hacked(
+                    args=prompt,
+                    latest_model="gpt-4o"
+                )
+                    return await message.edit_text(response)
+            except Exception as e:
+                await message.edit_text(str(e))
```

Comment:
change it to `message.edit_text(format_exc(e))`

Reply:
Done